gax-java/gax-httpjson/src/main/java/com/google/api/gax/httpjson/HttpJsonHeaderEnhancer.java

```diff
@@ -34,6 +34,7 @@
 
 /** Interface for functionality to enhance headers for an http-json call. */
 @BetaApi
+@FunctionalInterface
```

Comment:
I would not add `FunctionalInterface` to anything that is marked as `BetaApi`. Especially for things in httpjson folder.

Reply:
Yes

Reply:
done.